keyboards/skyloong/gk75/q1/info.json

```diff
@@ -0,0 +1,160 @@
+            {"matrix": [2, 0], "x": 0, "y": 26, "flags": 4},
+            {"matrix": [2, 1], "x": 22, "y": 26, "flags": 4},
+            {"matrix": [2, 2], "x": 37, "y": 26, "flags": 4},
+            {"matrix": [2, 3], "x": 51, "y": 26, "flags": 4},
+            {"matrix": [2, 4], "x": 66, "y": 26, "flags": 4},
+            {"matrix": [2, 5], "x": 81, "y": 26, "flags": 4},
+            {"matrix": [2, 6], "x": 96, "y": 26, "flags": 4},
+            {"matrix": [2, 7], "x": 110, "y": 26, "flags": 4},
+            {"matrix": [2, 8], "x": 125, "y": 26, "flags": 4},
+            {"matrix": [2, 9], "x": 140, "y": 26, "flags": 4},
+            {"matrix": [2, 10], "x": 154, "y": 26, "flags": 4},
+            {"matrix": [2, 11], "x": 169, "y": 26, "flags": 4},
+            {"matrix": [2, 12], "x": 184, "y": 26, "flags": 4},
+            {"matrix": [2, 13], "x": 198, "y": 26, "flags": 4},
+            {"matrix": [2, 14], "x": 224, "y": 26, "flags": 4},
+            {"matrix": [3, 0], "x": 0, "y": 38, "flags": 4},
+            {"matrix": [3, 1], "x": 26, "y": 38, "flags": 4},
+            {"matrix": [3, 2], "x": 40, "y": 38, "flags": 4},
+            {"matrix": [3, 3], "x": 55, "y": 38, "flags": 4},
+            {"matrix": [3, 4], "x": 70, "y": 38, "flags": 4},
+            {"matrix": [3, 5], "x": 86, "y": 38, "flags": 4},
+            {"matrix": [3, 6], "x": 99, "y": 38, "flags": 4},
+            {"matrix": [3, 7], "x": 114, "y": 38, "flags": 4},
+            {"matrix": [3, 8], "x": 129, "y": 38, "flags": 4},
+            {"matrix": [3, 9], "x": 143, "y": 38, "flags": 4},
+            {"matrix": [3, 10], "x": 158, "y": 38, "flags": 4},
+            {"matrix": [3, 11], "x": 173, "y": 38, "flags": 4},
+            {"matrix": [4, 14], "x": 209, "y": 26, "flags": 4},
+            {"matrix": [3, 12], "x": 187, "y": 38, "flags": 4},
+            {"matrix": [3, 13], "x": 198, "y": 38, "flags": 4},
+            {"matrix": [3, 14], "x": 224, "y": 38, "flags": 4},
+            {"matrix": [4, 0], "x": 0, "y": 51, "flags": 4},
+            {"matrix": [0, 1], "x": 18, "y": 51, "flags": 4},
+            {"matrix": [4, 1], "x": 33, "y": 51, "flags": 4},
+            {"matrix": [4, 2], "x": 48, "y": 51, "flags": 4},
+            {"matrix": [4, 3], "x": 62, "y": 51, "flags": 4},
+            {"matrix": [4, 4], "x": 77, "y": 51, "flags": 4},
+            {"matrix": [5, 6], "x": 92, "y": 51, "flags": 4},
+            {"matrix": [4, 6], "x": 106, "y": 51, "flags": 4},
+            {"matrix": [4, 7], "x": 121, "y": 51, "flags": 4},
+            {"matrix": [4, 8], "x": 136, "y": 51, "flags": 4},
+            {"matrix": [4, 9], "x": 151, "y": 51, "flags": 4},
+            {"matrix": [4, 10], "x": 165, "y": 51, "flags": 4},
+            {"matrix": [4, 11], "x": 180, "y": 51, "flags": 4},
+            {"matrix": [4, 13], "x": 209, "y": 51, "flags": 4},
+            {"matrix": [5, 0], "x": 0, "y": 64, "flags": 4},
+            {"matrix": [5, 1], "x": 18.4, "y": 64, "flags": 4},
```

Comment:
```suggestion
            {"matrix": [5, 1], "x": 18, "y": 64, "flags": 4},
```

Reply:
OK, It was be updated. Please review angin.

Reply:
OK, It was be updated. Please review angin.